### PR TITLE
"Tone down" node operator agreement language - Closes #2047.

### LINKED ISSUE #2047: Tone down staking and worklock messages?
Both WorkLock and Staking commands show the following text:
```
(...) Failure to keep your node online, or violation of re-encryption 
work orders will result in the loss of staked tokens as described
in the NuCypher slashing protocol.
```
Should it be toned down a little, specially after #1951 ?

### PR DESCRIPTION
**Pull request by KPrasch**

### CODE REVIEW

File: nucypher/cli/literature.py [hunk 1]
```diff
@@ -65,8 +65,9 @@
 - Agree to allow NuCypher network users to carry out uninterrupted re-encryption
   work orders at-will without interference.
 
-Failure to keep your node online, or violation of re-encryption work orders
-will result in the loss of staked tokens as described in the NuCypher slashing protocol.
+Failure to keep your node online or fulfill re-encryption work orders will result
+in loss of staked NU tokens as described in the NuCypher slashing protocol:
```
Comment (cygnusv) [inline nucypher/cli/literature.py:69]:
```suggestion
in loss of staked NU as described in the NuCypher slashing protocol:
```

**Comment by KPrasch** [comment]
Agree - Just trying to reference the open issue name.

**Comment by KPrasch** [comment]
I'm certainly open to suggestions, if this is not enough - @cygnusv?

File: nucypher/cli/literature.py [hunk 2]
```diff
@@ -462,19 +463,18 @@
 
 - By participating in NuCypher's WorkLock you are committing to operating a staking
   NuCypher node once the allocation period opens.
-
-- WorkLock allocation is provided in the form of a stake and will be locked for
-  the stake duration ({duration} periods). During this time, 
-  you are obligated to maintain a networked and available
-  NuCypher node bonded to the staker address {bidder_address}.
+    
+- Once claiming WorkLock tokens, you are obligated to maintain a networked and available
```
Comment (mswilkison) [inline nucypher/cli/literature.py:467]:
```suggestion
- Once claiming WorkLock NU, you are obligated to maintain a networked and available
```

**Pull request merged**